src/main/java/net/tirasa/connid/bundles/zimbra/ZimbraConnector.java

```diff
@@ -317,6 +318,16 @@ public Uid create(ObjectClass objClass, Set<Attribute> attrs, OperationOptions o
                         sp().addAlias(account, toObjectName(objClass, alias));
                     }
                 }
+                Attribute distributionListAttr = AttributeUtil.find("__DISTRIBUTION_LISTS__", attrs);
+                if (distributionListAttr != null && distributionListAttr.getValue() != null
+                        && !CollectionUtil.isEmpty(distributionListAttr.getValue())) {
```

Comment:
`CollectionUtil#isEmpty` will also check for null, you should remove `distributionListAttr.getValue() != null`